plugins/module_utils/scale.py

```diff
@@ -29,11 +29,16 @@
 
 try:
     import yaml
-    from openshift import watch
     from openshift.dynamic.client import ResourceInstance
-    from openshift.helper.exceptions import KubernetesException
-except ImportError as exc:
-    class KubernetesException(Exception):
+    from openshift.dynamic.exceptions import NotFoundError
+except ImportError:
+    pass
+try:
+    from openshift import watch
```

Comment:
I don't think we have a `watch` module in the openshift client anymore, we just make use of the `kubernetes` `watch` module instead. Only way to call a watch from the openshift client is the `DynamicClient.watch` method, though using `watch` from `kubernetes` should work fine.

Reply:
I'm moving this to a separate PR for scale improvements